config/samples/quotas/README.md

```diff
@@ -44,6 +44,19 @@ The fields above are defined at the top level CR and applied by default to all
 the `GlanceAPI` instances.
 It's not possible setup or override those values for each GlanceAPI instances
 independently.
+
+**Note:** When a registered limit is set to `0`, it means there is no limit
+enforced for that resource type. This allows unlimited consumption of the
+resource.
+
```

Comment:
I think this will convey wrong message. If imageCountTotal is 0 then user will not be able to create any image at all.
You should replicate the text from upstream documentation as is.

https://docs.openstack.org/glance/latest/admin/quotas.html#quota-strategies